scrubber/scrubberConfig.json

```diff
@@ -22,8 +24,28 @@
     },
     "auth": {
       "tagsToKeep": ["auth"]
+    },
+    "no-auth": {
+      "tagsToKeep": [],
+      "filesToDelete": [
+        "backend/typescript/services/implementations/userService.ts",
+        "backend/typescript/services/interfaces/userService.ts",
+        "backend/typescript/services/implementations/authService.ts",
+        "backend/typescript/services/interfaces/authService.ts",
+        "backend/typescript/rest/userRoutes.ts",
+        "backend/typescript/rest/authRoutes.ts",
+        "backend/typescript/middlewares/auth.ts",
+        "backend/typescript/middlewares/auth.graphql.ts",
```

Comment:
Just a note that these 2 files should be combined into one and have `rest` and `graphql` tags added, you can leave it for this PR though